In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
from pathlib import Path

import plotly.express as px
import plotly.graph_objects as go
from plotly.subplots import make_subplots

from sklearn.preprocessing import StandardScaler, OneHotEncoder, MinMaxScaler
from sklearn.decomposition import PCA
from sklearn.cluster import KMeans, DBSCAN
from sklearn.mixture import GaussianMixture
from sklearn.metrics import silhouette_score, calinski_harabasz_score, davies_bouldin_score

from sklearn.preprocessing import StandardScaler
from sklearn.metrics import silhouette_score, calinski_harabasz_score, davies_bouldin_score
from sklearn.decomposition import PCA
import warnings

from utils import *

In [2]:
df_sintomas_disc = pd.read_excel(r"data\Sintomas_disc_Infante_Juvenil_n=1558.xlsx")
df_etiquetas = pd.read_excel(r"data\preguntas_disc.xlsx")

In [3]:
q_list=question_list()

imprimir_etiquetas(q_list, df_etiquetas)


LISTADO DE ETIQUETAS (137 variables)
  1. [pso001] - Has friends
  2. [pso002] - Enjoys time with relatives of same age
  3. [pso003] - Nervous around new people in last year
  4. [pso004] - Nervous in a roup of children (party/school) in last year
  5. [pso005] - Nervous when doing things infront of other people
  6. [pga001] - Often worried before taking test/handing in assignment, past year
  7. [pga002] - Often seem worried before playing a sports game or doing an activity , pasy year
  8. [pga003] - Worried a lot about small mistakes made in projects or activities, past year
  9. [pga004] - Often worried about being on time in past year
 10. [pga005] - Often worried about having an illness in past year
 11. [pga023] - Type of person who is tense and finds it hard to relax
 12. [pga024] - In past year often been worried that they made a mistake/did something wrong
 13. [pga025] - In past year been worried that they made a fool of themselves infront of others
 14. [pga026] - Been v

In [4]:
df=df_sintomas_disc.copy()
final_df, prob_df=perform_lca_clustering(df, q_list, id_column='id')

_,_,_,_,_=analizar_distribucion(final_df, q_list)

ANÁLISIS DE CLASES LATENTES (LCA)

[1] Preparando datos: 1558 observaciones
[2] Procesando columnas de IMC...
    Transformando peso y estatura a categorías de IMC...
    ✓ IMC categorizado: ['IMC_pea002', 'IMC_pea003']

[3] Aplicando mapeo de códigos...
    ✓ Mapeo aplicado a 136 columnas

[4] Verificando tipos de datos...

[5] Análisis de valores faltantes (NaNs):
    pso001         :    1 NaNs (  0.06%)
    pso002         :    1 NaNs (  0.06%)
    pso003         :    1 NaNs (  0.06%)
    pso004         :    1 NaNs (  0.06%)
    pso005         :    1 NaNs (  0.06%)
    pga001         :   45 NaNs (  2.89%)
    pga002         :    2 NaNs (  0.13%)
    pga003         :    2 NaNs (  0.13%)
    pga004         :    2 NaNs (  0.13%)
    pga005         :    2 NaNs (  0.13%)
    pga023         :    2 NaNs (  0.13%)
    pga024         :    2 NaNs (  0.13%)
    pga025         :    2 NaNs (  0.13%)
    pga026         :    2 NaNs (  0.13%)
    pga027         :    2 NaNs (  0.13%)
    pga028      

# Perfil Típico de la muestra N = 1558

## Perfil Físico
- **IMC actual (IMC_pea002):**  
  Mayoritariamente **normal weight** (53.34%), pero con una proporción considerable de **underweight** (35.49%). Solo minorías en overweight (7.89%) y obese (3.08%).
  
- **IMC más bajo en el último año (IMC_pea003):**  
  Distribución casi igualada entre **normal weight** (46.92%) y **underweight** (44.61%). Esto sugiere que una parte significativa de la muestra ha tenido peso bajo recientemente.

---

## Síntomas Universales (prevalencia > 80%)
El perfil típico **casi siempre presenta**:
- **pso001:** Tiene amigos (**95.95%**).
- **psa011:** Ha estado separado de su figura de apego en el último año (**86.58%**).

---

## Síntomas Muy Comunes (prevalencia 61-80%)
- **pso002:** Disfruta del tiempo con compañeros/relaciones de su misma edad (**75.66%**).
- **psa006:** Fue invitado a pasar la noche fuera de casa (**62.75%**).

---

## Síntomas Moderadamente Frecuentes (prevalencia 41-60%)
El perfil típico **presenta varios de estos síntomas**:
- **pod002:** Discutía/replicaba a sus cuidadores (**55.78%**).
- **pod009:** Se sentía injustamente tratado y se enojaba (**54.18%**).
- **pad005:** Desorganizado (**51.16%**).
- **psa007:** Temía que algo malo le pasara a su figura de apego (**51.0%**).
- **pga004:** Preocupado por llegar a tiempo (**50.64%**).
- **pso005:** Nervioso al hacer cosas frente a otras personas (**48.68%**).
- **pmd003:** Irritable y se enojaba por pequeñas cosas (**48.59%**).
- **pmd018:** Dificultad para mantener la atención en tareas (**45.50%**).
- **psa012:** Estuvo fuera de casa varios días (**45.79%**).
- **pga001:** Se preocupaba mucho antes de exámenes/entregas (**44.61%**).
- **pmd001:** Períodos de tristeza/depresión (**42.87%**).

---

## Síntomas Ocasionales (prevalencia 21-40%)
El perfil típico **podría experimentar ocasionalmente**:
- **Consumo de sustancias:**
  - **Tabaco** (pni001): 20.12%
  - **Alcohol** (pal001): 24.55%
  
- **Síntomas internalizantes:**
  - **psa002:** Se sintió enfermo (39.70%)
  - **pga003:** Preocupación excesiva por pequeños errores (38.24%)
  - **pga024:** Preocupado por haber cometido errores (39.40%)
  - **pmd020:** Pensamientos sobre la muerte (30.33%)
  
- **Síntomas externalizantes:**
  - **pod001:** Perdía el temperamento (37.60%)
  - **pad001:** Problemas de atención sostenida (39.72%)
  - **pad006:** Dificultad para terminar tareas (37.15%)
  - **pad029:** Interrumpía a otros (41.65%)

---

## Síntomas Raros (prevalencia 0-20%)
El perfil típico **NO presenta** la mayoría de síntomas graves:
- **Consumo de drogas ilegales** : casi todos < 1%
- **Conducta antisocial grave** (robo con violencia, uso de armas): < 1-7%
- **Síntomas psicóticos** (alucinaciones, delirios): 0.96-11.63%
- **Ideación/conducta suicida**: 
  - pmd021 (ideas suicidas): 6.75%
  - pmd022 (intento suicida): 5.91%
- **Consumo de marihuana** (pmj001): 6.36%

---

## RESUMEN DEL Perfil TÍPICO

### **Características clave:**
1. **Perfil físico mixto:** Mayoría con peso normal, pero proporción alta con bajo peso actual o reciente.
2. **Altamente socializado:** Casi universalmente tiene amigos y disfruta de relaciones con pares.
3. **En transición hacia independencia:** Separación frecuente de figuras de apego, noches fuera de casa común.
4. **Síntomas internalizantes moderados:**
   - **Ansiedad de separación** significativa
   - **Ansiedad social y de rendimiento**
   - **Síntomas depresivos** (tristeza, irritabilidad)
5. **Síntomas externalizantes leves-moderados:**
   - **Oposicionismo** (discute con cuidadores, se siente injustamente tratado)
   - **Problemas atencionales y de organización** (TDAH leve)
6. **Consumo de sustancias bajo-moderado:**
   - Alcohol y tabaco presentes pero no mayoritarios
   - Drogas ilegales muy raras
7. **Bajo riesgo de psicopatología grave:**
   - Psicosis muy infrecuente
   - Conducta antisocial grave poco común
   - Ideación suicida presente pero en minoría

### **Riesgo clínico:**
**Moderado-bajo.** La mayoría de síntomas son leves a moderados y probablemente dentro del rango del desarrollo normal infanto-juvenil. Sin embargo, la combinación de síntomas depresivos, ansiedad y problemas atencionales podría requerir evaluación en algunos casos.


In [6]:
ids_filtrados = final_df.loc[final_df['lca_k3'] == 1, 'id']

df_norm = df_sintomas_disc[df_sintomas_disc['id'].isin(ids_filtrados)]
q_list=question_list()

df=df_norm.copy()
final_df_norm, prob_df_norm=perform_lca_clustering(df, q_list, id_column='id')

_,_,_,_,_=analizar_distribucion(final_df_norm, q_list)

resultados = analisis_estadistico_clusters(
    final_df=final_df_norm,          
    id_column='id',              
    cluster_prefix='lca_k',     
    umbral_diferencia=0.10,      
    top_n_variables=25,         
    umbral_asociacion=0.15      
)

ANÁLISIS DE CLASES LATENTES (LCA)

[1] Preparando datos: 968 observaciones
[2] Procesando columnas de IMC...
    Transformando peso y estatura a categorías de IMC...
    ✓ IMC categorizado: ['IMC_pea002', 'IMC_pea003']

[3] Aplicando mapeo de códigos...
    ✓ Mapeo aplicado a 136 columnas

[4] Verificando tipos de datos...

[5] Análisis de valores faltantes (NaNs):
    pga001         :   27 NaNs (  2.79%)
    psa002         :   27 NaNs (  2.79%)
    pmd035         :   23 NaNs (  2.38%)
    IMC_pea002     :    1 NaNs (  0.10%)
    IMC_pea003     :    1 NaNs (  0.10%)

    Total: 79 NaNs en 131,648 celdas (0.06%)

[6] Preparando datos para LCA (todas variables como categóricas)...
    ✓ Datos preparados: 968 × 136
    ✓ Columnas codificadas: 136

[7] Realizando Análisis de Clases Latentes...

    k = 2 clases latentes...
      ✓ BIC: -304961.54, AIC: -307618.54
      ✓ Certeza promedio: 1.000
      ✓ Distribución:
          Clase 0: 770 (79.5%)
          Clase 1: 198 (20.5%)

    k = 3 c

# Perfil Típico Normativo

## Perfil Físico
- **IMC actual (IMC_pea002):**  
  Distribución casi equilibrada entre **normal weight** (49.59%) y **underweight** (41.01%). Una minoría tiene sobrepeso (6.20%) u obesidad (3.10%).
  
- **IMC más bajo en el último año (IMC_pea003):**  
  Predomina el **underweight** (48.76%), seguido de **normal weight** (43.08%). Esto indica que una proporción significativa ha tenido peso bajo recientemente.

---

## Síntomas Universales (prevalencia > 80%)
El perfil típico **casi siempre presenta**:
- **pso001:** Tiene amigos (**95.66%**).
- **psa011:** Ha estado separado de su figura de apego en el último año (**84.61%**).

---

## Síntomas Muy Comunes (prevalencia 61-80%)
- **pso002:** Disfruta del tiempo con compañeros/relaciones de su misma edad (**77.58%**).

---

## Síntomas Moderadamente Frecuentes (prevalencia 41-60%)
El perfil típico **presenta varios de estos síntomas**:
- **psa006:** Fue invitado a pasar la noche fuera de casa (**56.92%**).
- **pga004:** Preocupado por llegar a tiempo (**51.45%**).
- **psa007:** Temía que algo malo le pasara a su figura de apego (**46.07%**).
- **pod009:** Se sentía injustamente tratado y se enojaba (**44.42%**).
- **pod002:** Discutía/replicaba a sus cuidadores (**43.49%**).
- **pso005:** Nervioso al hacer cosas frente a otras personas (**43.29%**).
- **pad005:** Desorganizado (**43.08%**).
- **pga001:** Se preocupaba mucho antes de exámenes/entregas (**41.66%**).
- **psa012:** Estuvo fuera de casa varios días (**40.29%**).

---

## Síntomas Ocasionales (prevalencia 21-40%)
El perfil típico **podría experimentar ocasionalmente**:
- **Síntomas internalizantes:**
  - **pmd003:** Irritable y se enojaba por pequeñas cosas (**39.36%**)
  - **psa005:** A veces no podía dormir sin la figura de apego presente (**38.22%**)
  - **pad004:** No podía mantener la atención con distractores (**37.91%**)
  - **psa002:** Se sintió enfermo (**35.07%**)
  - **pmd001:** Períodos de tristeza/depresión (**34.40%**)
  - **pmd018:** Dificultad para mantener la atención (**34.81%**)
  - **pga003:** Preocupación por pequeños errores (**35.54%**)

- **Consumo de sustancias:**
  - **Alcohol** (pal001): 14.57%
  - **Tabaco** (pni001): 10.54%
  - **Marihuana** (pmj001): 2.27%

---

## Síntomas Muy Raros (prevalencia 0-20%)
El perfil típico **NO presenta** la mayoría de síntomas graves:
- **Ideación/conducta suicida**: 0% (pmd021, pmd022)
- **Síntomas psicóticos graves** (psz012, psz013, psz014): 0%
- **Conducta antisocial grave**: 0% (robo con violencia, uso de armas, vandalismo grave)
- **Consumo de drogas ilegales**: 0% (estimulantes, sedantes, cocaína, heroína, etc.)
- **Problemas graves de conducta**: muy bajos (<10% para la mayoría)

---

## RESUMEN DEL Perfil TÍPICO

### **Características clave:**
1. **Perfil físico con tendencia al bajo peso:** Casi la mitad tiene o ha tenido bajo peso recientemente.
2. **Socialización excelente:** Alta prevalencia de amigos y disfrute con pares.
3. **Ansiedad de separación significativa:** Separación frecuente de figura de apego, preocupación por su bienestar.
4. **Síntomas internalizantes moderados-bajos:**
   - Ansiedad de separación presente
   - Preocupaciones por rendimiento y puntualidad
   - Síntomas depresivos leves (tristeza, irritabilidad)
5. **Síntomas externalizantes mínimos:**
   - Oposicionismo leve (discute, se siente injustamente tratado)
   - Problemas atencionales moderados
   - Casi ausencia de conducta antisocial
6. **Consumo de sustancias muy bajo:**
   - Alcohol y tabaco en proporciones bajas
   - Drogas ilegales prácticamente inexistentes
7. **Ausencia de psicopatología grave:**
   - Sin ideación/conducta suicida
   - Sin síntomas psicóticos
   - Sin conductas antisociales graves
     
### **Riesgo clínico:**
**Muy bajo.** La mayoría de síntomas son leves y probablemente dentro del desarrollo normal. La ausencia total de ideación suicida y conductas graves es notable.



# Análisis de Jerarquización de Clases (LCA: k=2 a k=6)

## Evolución Jerárquica de las Clases para Grupo Normativo:


| k de Origen | Clase de Origen | Total Obs. | Destino Principal (k+1) [% → Clase(Obs.)] | Destino Secundario (k+1) [% → Clase(Obs.)] | Otros Destinos |
| :---: | :---: | :---: | :---: | :---: | :---: |
| **2 → 3** | 0 | 770 | 49.2% → **2** (379) | 47.4% → **1** (365) | 3.4% → 0 (26) |
| | 1 | 198 | 71.2% → **0** (141) | 28.8% → **1** (57) | - |
| **3 → 4** | 0 | 167 | 58.1% → **3** (97) | 28.7% → **2** (48) | 13.2% → 1 (22) |
| | 1 | 422 | 73.0% → **0** (308) | 27.0% → **1** (114) | - |
| | 2 | 379 | 60.7% → **0** (230) | 24.5% → **3** (93) | 14.8% → 1 (56) |
| **4 → 5** | 0 | 538 | 54.1% → **3** (291) | 43.5% → **0** (234) | 2.2% → 1 (12) |
| | 1 | 192 | 49.0% → **3** (94) | 21.4% → **1** (41) | 29.7% (0,4,2) |
| | 2 | 48 | 79.2% → **2** (38) | 20.8% → **1** (10) | - |
| | 3 | 190 | 46.8% → **4** (89) | 38.4% → **1** (73) | 14.7% → 0 (28) |
| **5 → 6** | 0 | 290 | 50.0% → **0** (145) | 40.0% → **5** (116) | 9.9% (2,3,4) |
| | 1 | 136 | 50.0% → **5** (68) | 22.8% → **2** (31) | 27.2% (4,3,1) |
| | 2 | 43 | 74.4% → **1** (32) | 18.6% → **4** (8) | 7.0% (2,3) |
| | 3 | 385 | 67.0% → **0** (258) | 31.2% → **2** (120) | 1.8% (5,4) |
| | 4 | 114 | 46.5% → **4** (53) | 33.3% → **3** (38) | 20.2% (2,1) |

*   **k=2, Clase 0**: El grupo mayoritario original se fractura casi equitativamente en las futuras Clase 2 (49.2%) y Clase 1 (47.4%) de k=3. Esta división origina dos ramas principales que evolucionan de forma paralela: una que se consolida en la **Clase 0** de k=5 y luego en la **Clase 0** de k=6, y otra que termina en la **Clase 5** de k=6.
*   **k=2, Clase 1**: El grupo minoritario se traslada mayoritariamente (71.2%) a la **Clase 0** de k=3. Esta clase luego da origen a la **Clase 3** de k=4, que se divide para formar las **Clases 4 y 1** de k=5. Finalmente, estos grupos se redistribuyen en las **Clases 4, 3, 1 y 2** de k=6, mostrando una evolución más dispersa.
*   **Estabilidad**: No se observa una trayectoria única con una estabilidad muy alta (>90%). En cambio, la evolución es más compleja, con divisiones significativas y mezclas en cada transición. La mayor estabilidad se ve en transiciones puntuales, como la de la **Clase 2 de k=4**, que retiene el 79.2% de sus observaciones en el **Clúster 2 de k=5**, o la del **Clúster 0 de k=5**, donde la mitad permanece en el **Clúster 0 de k=6**. La transición de k=5 a k=6 es particularmente mixta, con la mayoría de las clases de origen dividiéndose en dos o más destinos principales.

# Clasificación LCA para grupo Normativo

## Variables más discriminatorias (k = 3)

| Variable | Cramér's V | p-value   | χ²  | Discriminación |
|----------|------------|-----------|-----|----------------|
| pal001   | 0.5066     | 1.12e-54  | 248 | 1.000          |
| pad025   | 0.4078     | 1.10e-35  | 161 | 1.000          |
| pcd012   | 0.3984     | 4.41e-34  | 154 | 1.000          |
| pcd027   | 0.3350     | 2.57e-24  | 109 | 1.000          |
| pmj001   | 0.3340     | 3.58e-24  | 108 | 1.000          |
| pcd003   | 0.3340     | 3.58e-24  | 108 | 1.000          |
| pni001   | 0.3327     | 3.21e-45  | 214 | 0.998          |
| psa005   | 0.3251     | 6.16e-23  | 102 | 1.000          |
| psz009   | 0.3155     | 1.21e-21  | 96  | 1.000          |
| pcd014   | 0.3122     | 3.21e-21  | 94  | 1.000          |
| pmd003   | 0.3076     | 1.29e-20  | 92  | 1.000          |
| pmd012   | 0.3034     | 4.52e-20  | 89  | 1.000          |
| pmd015   | 0.3029     | 5.16e-20  | 89  | 1.000          |
| pmd016   | 0.2899     | 2.19e-18  | 81  | 1.000          |
| pmd020   | 0.2870     | 4.83e-18  | 80  | 1.000          |
| pmd018   | 0.2776     | 6.29e-17  | 75  | 1.000          |
| pmd001   | 0.2771     | 7.31e-17  | 74  | 1.000          |
| psz011   | 0.2748     | 1.35e-16  | 73  | 1.000          |
| pmd017   | 0.2737     | 1.79e-16  | 73  | 1.000          |
| pad001   | 0.2727     | 2.34e-16  | 72  | 1.000          |
| pmd010   | 0.2707     | 3.99e-16  | 71  | 1.000          |
| psa007   | 0.2705     | 4.20e-16  | 71  | 1.000          |
| psa010   | 0.2686     | 6.90e-16  | 70  | 1.000          |
| pmd008   | 0.2684     | 7.26e-16  | 70  | 1.000          |
| pga028   | 0.2661     | 1.19e-28  | 137 | 0.999          |

## Hallazgos principales:

### Variable más discriminante:
- **`pal001`** – *"Alguna vez en la vida haber tomado al menos un trago de alcohol (excluyendo sorbos de otros)"*
    - **Cramér's V = 0.5066** (el más alto)
    - **p = 1.12e-54**

### Dimensiones temáticas predominantes:

#### Consumo de sustancias:
- `pal001` – Alguna vez en la vida haber tomado al menos un trago de alcohol (excluyendo sorbos de otros)
- `pmj001` – Alguna vez en la vida usado marihuana
- `pni001` – Alguna vez fumado un cigarrillo

#### Conducta antisocial, disruptiva y problemas de atención:
- `pad025` – A menudo trepaba en cosas/correa cuando no debía en el último año
- `pcd012` – Alguna vez se metió en problemas por quedarse fuera 2 horas después del toque de queda
- `pcd027` – Alguna vez estado en una pelea física en la que alguien resultó herido/podría haberse herido
- `pcd003` – Alguna vez robó de alguien cuando no estaba/nadie miraba
- `pcd014` – Alguna vez mintió para obtener dinero o algo más
- `pad001` – Problemas para mantener la mente en la tarea por más de un período corto de tiempo

#### Sintomatología internalizante (depresiva, ansiosa y somática):
- `pmd003` – A menudo malhumorado e irritable, y pequeñas cosas creaban enojo en el último año
- `pmd012` – Menos energía en el último año
- `pmd015` – Se culpó a sí mismo por cosas malas que sucedieron en el último año
- `pmd016` – Dijo que no podía hacer nada bien/no era tan bonito/inteligente como otros en el último año
- `pmd020` – A menudo pensó en la muerte/gente muriendo/estar muerto en el último año
- `pmd018` – Problemas para mantener la mente en la escuela/otras cosas en el último año
- `pmd001` – Un tiempo en el último año cuando a menudo parecía muy deprimido
- `pmd017` – No podía pensar tan rápido o claramente como de costumbre en el último año
- `pmd010` – Parecía hacer cosas como caminar/hablar mucho más lento en el último año
- `pmd008` – Problemas para dormir en el último año
- `psa005` – A veces no podía conciliar el sueño sin la figura de apego presente en el último año
- `psa007` – Temía que algo le pasara a la figura de apego en el último año
- `psa010` – Nervioso/alterado lejos de la figura de apego en el último año
- `pga028` – A menudo tuvo dolores de estómago en el último año

#### Síntomas psicóticos (paranoia/ideas de referencia):
- `psz009` – Creer que la gente te espiaba
- `psz011` – Creer que alguien conspiraba contra ti/intentaba hacerte daño

## Categorías Sub y Sobre-Representadas

### Clase 0 (k = 3)
167 observaciones (17.3%)
  
  ---
  Variables distintivas:

     1. pal001:
        SOBRE 1: 'SI' - 51% (86/167) vs 15% (141/968) global, +37pp
        SUB 1: 'NO' - 49% (81/167) vs 85% (827/968) global, -37pp

     2. pni001:
        SOBRE 1: 'SI' - 41% (68/167) vs 11% (102/968) global, +30pp
        SUB 1: 'NO' - 59% (98/167) vs 89% (865/968) global, -31pp

     3. psa007:
        SOBRE 1: 'SI' - 74% (123/167) vs 46% (446/968) global, +28pp
        SUB 1: 'NO' - 26% (44/167) vs 54% (522/968) global, -28pp

     4. pmd018:
        SOBRE 1: 'SI' - 56% (94/167) vs 35% (337/968) global, +21pp
        SUB 1: 'NO' - 44% (73/167) vs 65% (631/968) global, -21pp

     5. pmd016:
        SOBRE 1: 'SI' - 43% (71/167) vs 21% (205/968) global, +21pp
        SUB 1: 'NO' - 57% (96/167) vs 79% (763/968) global, -21pp

     6. pmd012:
        SOBRE 1: 'SI' - 40% (67/167) vs 19% (185/968) global, +21pp
        SUB 1: 'NO' - 60% (100/167) vs 81% (783/968) global, -21pp

     7. pmd001:
        SOBRE 1: 'SI' - 54% (91/167) vs 34% (333/968) global, +20pp
        SUB 1: 'NO' - 46% (76/167) vs 66% (635/968) global, -20pp

     8. pmd008:
        SOBRE 1: 'SI' - 45% (75/167) vs 26% (247/968) global, +19pp
        SUB 1: 'NO' - 55% (92/167) vs 74% (721/968) global, -19pp

     9. pmd017:
        SOBRE 1: 'SI' - 34% (56/167) vs 14% (137/968) global, +19pp
        SUB 1: 'NO' - 66% (111/167) vs 86% (831/968) global, -19pp

    10. pmd015:
        SOBRE 1: 'SI' - 35% (59/167) vs 16% (157/968) global, +19pp
        SUB 1: 'NO' - 65% (108/167) vs 84% (811/968) global, -19pp

    11. pmd003:
        SOBRE 1: 'SI' - 57% (96/167) vs 39% (381/968) global, +18pp
        SUB 1: 'NO' - 43% (71/167) vs 61% (587/968) global, -18pp

    12. pmd020:
        SOBRE 1: 'SI' - 39% (65/167) vs 22% (212/968) global, +17pp
        SUB 1: 'NO' - 61% (102/167) vs 78% (756/968) global, -17pp

    13. pcd027:
        SOBRE 1: 'SI' - 23% (38/167) vs 6% (58/968) global, +17pp
        SUB 1: 'NO' - 77% (129/167) vs 94% (910/968) global, -17pp

    14. pcd012:
        SOBRE 1: 'SI' - 19% (31/167) vs 3% (31/968) global, +15pp
        SUB 1: 'NO' - 81% (136/167) vs 97% (937/968) global, -15pp

    15. pad001:
        SOBRE 1: 'SI' - 47% (78/167) vs 32% (305/968) global, +15pp
        SUB 1: 'NO' - 53% (89/167) vs 68% (663/968) global, -15pp

    16. psz009:
        SOBRE 1: 'SI' - 20% (34/167) vs 5% (52/968) global, +15pp
        SUB 1: 'NO' - 80% (133/167) vs 95% (916/968) global, -15pp

    17. pcd014:
        SOBRE 1: 'SI' - 20% (33/167) vs 5% (50/968) global, +15pp
        SUB 1: 'NO' - 80% (134/167) vs 95% (918/968) global, -15pp

    18. pga028:
        SOBRE 1: 'SI' - 28% (47/167) vs 15% (148/968) global, +13pp
        SUB 1: 'NO' - 72% (120/167) vs 85% (819/968) global, -13pp

    19. pmj001:
        SOBRE 1: 'SI' - 13% (22/167) vs 2% (22/968) global, +11pp
        SUB 1: 'NO' - 87% (145/167) vs 98% (946/968) global, -11pp

    20. pcd003:
        SOBRE 1: 'SI' - 13% (22/167) vs 2% (22/968) global, +11pp
        SUB 1: 'NO' - 87% (145/167) vs 98% (946/968) global, -11pp

    21. pmd010:
        SOBRE 1: 'SI' - 19% (31/167) vs 8% (76/968) global, +11pp
        SUB 1: 'NO' - 81% (136/167) vs 92% (892/968) global, -11pp

    22. psa010:
        SOBRE 1: 'SI' - 30% (50/167) vs 20% (191/968) global, +10pp
        SUB 1: 'NO' - 70% (117/167) vs 80% (777/968) global, -10pp

#### Perfil general:
La **Clase 0** representa un **subgrupo minoritario (17.3%)** dentro de la muestra normativa y se caracteriza por un **perfil de mayor reactividad emocional y experimentación temprana**, mostrando una elevación simultánea en síntomas de ansiedad, estado de ánimo depresivo y ciertas conductas externalizantes en comparación con la media del grupo.

#### Hallazgos principales:

##### **Elevación en síntomas de ansiedad por separación y somatización:**
- **`psa007`**: 74%  temía que algo le pasara a la figura de apego (vs 46% global) → **+28pp**
- **`pga028`**: 28%  tuvo dolores de estómago frecuentes (vs 15% global) → **+13pp**
- **`psa010`**: 30%  se sintió nervioso/alterado lejos de la figura de apego (vs 20% global) → **+10pp**

##### **Elevación en síntomas depresivos y cognitivos:**
- **`pmd018`**: 56%  tuvo problemas para mantener la mente en la escuela/otras cosas (vs 35% global) → **+21pp**
- **`pmd016`**: 43%  dijo que no podía hacer nada bien/no era tan capaz como otros (vs 21% global) → **+21pp**
- **`pmd012`**: 40%  tuvo menos energía (vs 19% global) → **+21pp**
- **`pmd001`**: 54%  pareció muy deprimido/alterado en algún período (vs 34% global) → **+20pp**
- **`pmd008`**: 45%  tuvo problemas para dormir (vs 26% global) → **+19pp**
- **`pmd015`**: 35%  se culpó a sí mismo por cosas malas (vs 16% global) → **+19pp**
- **`pmd003`**: 57%  estuvo malhumorado e irritable (vs 39% global) → **+18pp**
- **`pmd020`**: 39%  pensó en la muerte/gente muriendo (vs 22% global) → **+17pp**
- **`pmd010`**: 19%  pareció hacer cosas (caminar/hablar) mucho más lento (vs 8% global) → **+11pp**
- **`pmd017`**: 34%  no podía pensar tan rápido o claramente (vs 14% global) → **+19pp**

##### **Consumo de sustancias:**
- **`pal001`**: 51%  ha tomado al menos un trago de alcohol (vs 15% global) → **+37pp**
- **`pni001`**: 41%  ha fumado cigarrillos (vs 11% global) → **+30pp**
- **`pmj001`**: 13%  ha usado marihuana (vs 2% global) → **+11pp**

##### **Presencia de conductas externalizantes específicas:**
- **`pcd027`**: 23%  ha estado en una pelea física con heridos (vs 6% global) → **+17pp**
- **`pcd012`**: 19%  se ha metido en problemas por llegar tarde (toque de queda) (vs 3% global) → **+15pp**
- **`pcd014`**: 20%  ha mentido para obtener dinero o algo (vs 5% global) → **+15pp**
- **`pcd003`**: 13%  ha robado cuando no había nadie (vs 2% global) → **+11pp**

##### **Experiencias de desconfianza interpersonal:**
- **`psz009`**: 20%  ha creído que la gente les espiaba (vs 5% global) → **+15pp**

##### **Dificultades atencionales:**
- **`pad001`**: 47%  tiene problemas para mantener la mente en la tarea (vs 32% global) → **+15pp**

#### Síntesis interpretativa:

La **Clase 0** constituye un **subgrupo diferenciado (17.3%)** dentro de la variación normativa, caracterizado por un **patrón de reactividad emocional elevada y experimentación**. Este perfil sugiere:

1.  **Mayor internalización**: Combinación de síntomas ansiosos (especialmente de separación), quejas somáticas y un estado de ánimo depresivo con autocrítica.
2.  **Policonsumo**: Aresencia frecuente en el consumo de alcohol, tabaco y, en menor medida, marihuana.
3.  **Externalización**: Presencia de algunas conductas como peleas, transgresiones de normas y mentiras-
4.  **Desconfianza interpersonal**: Mayor tendencia a experiencias paranoicas leves (sentirse espiado).

Este patrón refleja un **estilo de afrontamiento más reactivo** dentro del espectro normal, donde la angustia internalizada se manifiesta junto con una búsqueda de experiencias (substancias) y cierta dificultad en la regulación de la conducta y la atención. No representa un cuadro clínico, sino una variante dentro de la población general con mayor vulnerabilidad emocional.  

### Clase 1 (k = 3)

  
  422 observaciones (43.6%)
  
  ---
  Variables distintivas:

     1. pmd003:
        SOBRE 1: 'NO' - 77% (327/422) vs 61% (587/968) global, +17pp
        SUB 1: 'SI' - 23% (95/422) vs 39% (381/968) global, -17pp

     2. pad025:
        SOBRE 1: 'NO' - 100% (422/422) vs 84% (815/968) global, +16pp

     3. pga028:
        SOBRE 1: 'NO' - 100% (422/422) vs 85% (819/968) global, +15pp

     4. pad001:
        SOBRE 1: 'NO' - 83% (349/422) vs 68% (663/968) global, +14pp
        SUB 1: 'SI' - 17% (73/422) vs 32% (305/968) global, -14pp

     5. psa005:
        SOBRE 1: 'NO' - 76% (320/422) vs 62% (598/968) global, +14pp
        SUB 1: 'SI' - 24% (102/422) vs 38% (370/968) global, -14pp

     6. pmd001:
        SOBRE 1: 'NO' - 80% (336/422) vs 66% (635/968) global, +14pp
        SUB 1: 'SI' - 20% (86/422) vs 34% (333/968) global, -14pp

     7. pmd018:
        SOBRE 1: 'NO' - 79% (333/422) vs 65% (631/968) global, +14pp
        SUB 1: 'SI' - 21% (89/422) vs 35% (337/968) global, -14pp

     8. pmd020:
        SOBRE 1: 'NO' - 91% (384/422) vs 78% (756/968) global, +13pp
        SUB 1: 'SI' - 9% (38/422) vs 22% (212/968) global, -13pp

     9. psa010:
        SOBRE 1: 'NO' - 92% (390/422) vs 80% (777/968) global, +12pp
        SUB 1: 'SI' - 8% (32/422) vs 20% (191/968) global, -12pp

    10. pmd008:
        SOBRE 1: 'NO' - 86% (365/422) vs 74% (721/968) global, +12pp
        SUB 1: 'SI' - 14% (57/422) vs 26% (247/968) global, -12pp

    11. pmd012:
        SOBRE 1: 'NO' - 93% (391/422) vs 81% (783/968) global, +12pp
        SUB 1: 'SI' - 7% (31/422) vs 19% (185/968) global, -12pp

    12. pmd016:
        SOBRE 1: 'NO' - 90% (381/422) vs 79% (763/968) global, +11pp
        SUB 1: 'SI' - 10% (41/422) vs 21% (205/968) global, -11pp

    13. pmd015:
        SOBRE 1: 'NO' - 95% (401/422) vs 84% (811/968) global, +11pp

    14. psa007:
        SOBRE 1: 'NO' - 65% (273/422) vs 54% (522/968) global, +11pp
        SUB 1: 'SI' - 35% (149/422) vs 46% (446/968) global, -11pp

#### Perfil general:
La **Clase 1** representa el **subgrupo más grande (43.6%)** dentro de la muestra normativa y se caracteriza por un **perfil de notable estabilidad emocional y adaptación**, mostrando significativamente **menos síntomas internalizantes, externalizantes y somáticos** que la media del grupo total.

#### Hallazgos principales:

##### **Baja sintomatología depresiva y cognitiva:**
- **`pmd003`**: 77% No estuvo malhumorado e irritable (vs 61% global) → **+17pp**
- **`pmd001`**: 80% No pareció muy deprimido/alterado en algún período (vs 66% global) → **+14pp**
- **`pmd018`**: 79% No tuvo problemas para mantener la mente en la escuela/otras cosas (vs 65% global) → **+14pp**
- **`pmd020`**: 91% No pensó en la muerte/gente muriendo (vs 78% global) → **+13pp**
- **`pmd012`**: 93% No tuvo menos energía (vs 81% global) → **+12pp**
- **`pmd016`**: 90% No dijo que no podía hacer nada bien/no era tan capaz como otros (vs 79% global) → **+11pp**
- **`pmd015`**: 95% No se culpó a sí mismo por cosas malas (vs 84% global) → **+11pp**
- **`pmd008`**: 86% No tuvo problemas para dormir (vs 74% global) → **+12pp**

##### **Baja ansiedad por separación y somatización:**
- **`psa005`**: 76% No necesitaba la figura de apego presente para dormir (vs 62% global) → **+14pp**
- **`psa010`**: 92% No se sintió nervioso/alterado lejos de la figura de apego (vs 80% global) → **+12pp**
- **`psa007`**: 65% No temía que algo le pasara a la figura de apego (vs 54% global) → **+11pp**
- **`pga028`**: 100% No tuvo dolores de estómago frecuentes (vs 85% global) → **+15pp**

##### **Baja conducta disruptiva e hiperactiva:**
- **`pad025`**: 100% No trepaba en cosas/correa cuando no debía (vs 84% global) → **+16pp**

##### **Menos dificultades atencionales:**
- **`pad001`**: 83% No tiene problemas para mantener la mente en la tarea (vs 68% global) → **+15pp**

#### Síntesis interpretativa:

La **Clase 1** constituye el **subgrupo mayoritario y más adaptado (43.6%)** dentro de la variación normativa, caracterizado por un **patrón de notable estabilidad y resiliencia emocional**. Este perfil sugiere:

1.  **Excelente regulación emocional**: Mínima presencia de síntomas depresivos como irritabilidad, tristeza, fatiga, insomnio, autocrítica o pensamientos negativos.
2.  **Independencia emocional segura**: Baja ansiedad de separación, indicando vínculos de apego saludables y capacidad para funcionar autónomamente.
3.  **Baja reactividad somática**: Ausencia completa de quejas somáticas frecuentes como dolores de estómago.
4.  **Buena autorregulación conductual**: Sin comportamientos hiperactivos/inapropiados y menores dificultades atencionales.

Este patrón refleja un **estilo de afrontamiento estable y adaptativo** dentro del espectro normal, donde predominan la regulación emocional efectiva, la independencia segura y la ausencia de malestar significativo. Representa el perfil de mayor bienestar psicológico dentro de la distribución normativa de la muestra.

### Clase 2 (k = 3)
 379 observaciones (39.2%)

  ---
  Variables distintivas:

     1. psa005:
        SOBRE 1: 'SI' - 58% (219/379) vs 38% (370/968) global, +20pp
        SUB 1: 'NO' - 42% (160/379) vs 62% (598/968) global, -20pp

     2. pad025:
        SOBRE 1: 'SI' - 33% (124/379) vs 16% (153/968) global, +17pp
        SUB 1: 'NO' - 67% (255/379) vs 84% (815/968) global, -17pp

     3. pal001:
        SOBRE 1: 'NO' - 100% (379/379) vs 85% (827/968) global, +15pp

     4. pga028:
        SOBRE 1: 'SI' - 27% (101/379) vs 15% (148/968) global, +11pp
        SUB 1: 'NO' - 73% (277/379) vs 85% (819/968) global, -12pp

     5. pmd003:
        SOBRE 1: 'SI' - 50% (190/379) vs 39% (381/968) global, +11pp
        SUB 1: 'NO' - 50% (189/379) vs 61% (587/968) global, -11pp

     6. pni001:
        SOBRE 1: 'NO' - 100% (379/379) vs 89% (865/968) global, +11pp

#### Perfil general:
La **Clase 2** representa un **subgrupo considerable (39.2%)** dentro de la muestra normativa y se caracteriza por un **perfil de mayor ansiedad de separación y reactividad somática**, junto con una **notable abstinencia de sustancias**, mostrando una combinación de internalización emocional y cierta inquietud motora en comparación con la media del grupo.

#### Hallazgos principales:

##### **Elevada ansiedad de separación:**
- **`psa005`**: 58% no ha podido conciliar el sueño sin la figura de apego presente (vs 38% global) → **+20pp**

##### **Mayor reactividad somática:**
- **`pga028`**: 27% tuvo dolores de estómago frecuentes (vs 15% global) → **+11pp**

##### **Abstinencia de sustancias:**
- **`pal001`**: 100% No ha bebido al menos un trago de alcohol (vs 85% global) → **+15pp**
- **`pni001`**: 100% No ha fumado cigarrillos (vs 89% global) → **+11pp**

##### **Mayor inquietud motora e irritabilidad:**
- **`pad025`**: 33% trepaba en cosas/correa cuando no debía (vs 16% global) → **+17pp**
- **`pmd003`**: 50% estuvo malhumorado e irritable (vs 39% global) → **+11pp**

#### Síntesis interpretativa:

La **Clase 2** constituye un **subgrupo significativo (39.2%)** dentro de la variación normativa, caracterizado por un **patrón de apego ansioso con abstinencia conductual**. Este perfil sugiere:

1.  **Apego ansioso**: Mayor necesidad de proximidad con la figura de apego para la regulación emocional, especialmente en situaciones vulnerables como el sueño.
2.  **Somatización reactiva**: Tendencia a expresar el malestar emocional a través de síntomas físicos como dolores gastrointestinales.
3.  **Abstinencia conductual**: Evitación completa del consumo de alcohol y tabaco, lo que podría reflejar mayor cautela, adherencia a normas o menor búsqueda de sensaciones.
4.  **Inquietud e irritabilidad**: Elevación en conductas motoras inapropiadas y en irritabilidad.

Este patrón refleja un **estilo de afrontamiento más cauteloso y dependiente** dentro del espectro normal, donde la ansiedad de separación se manifiesta junto con quejas somáticas y una evitación clara de conductas de riesgo (consumo), combinado con cierta dificultad en la regulación motora y emocional. Representa una variante normativa con mayor vulnerabilidad ansiosa pero con protectores conductuales evidentes.

## Síntesis de Diferenciación de Clases (k=3)

| **Criterio** | **Clase 0 (Reactividad y Experimentación)** | **Clase 1 (Estabilidad y Adaptación)** | **Clase 2 (Apego Ansioso y Cautela)** |
| :--- | :--- | :--- | :--- |
| **Tamaño de la Clase** | 167 observaciones (17.3%) | 422 observaciones (43.6%) | 379 observaciones (39.2%) |
| **Patrón Principal** | **Perfil de reactividad elevada.** Combinación normativa de mayor labilidad emocional (ansiedad, síntomas depresivos leves), experimentación con sustancias y conductas de oposición leves. | **Perfil de ajuste óptimo.** Notable estabilidad emocional, baja sintomatología en todos los dominios y ausencia de conductas de riesgo. | **Perfil de cautela e internalización.** Ansiedad de separación marcada, abstinencia total de sustancias, con tendencia a la somatización y cierta inquietud. |
| **Núcleo del Perfil** | **Vulnerabilidad emocional con exploración.** Se caracteriza por una **elevación simultánea** en múltiples dominios del malestar normativo, mostrando un estilo de afrontamiento más reactivo y exploratorio. | **Resiliencia y regulación efectiva.** Se caracteriza por la **sub-representación generalizada** de dificultades, representando el polo de mayor adaptación y recursos dentro de la variación normal. | **Apego preocupado con evitación.** La angustia se canaliza hacia la preocupación por la separación y el cuerpo, coexistiendo con una fuerte evitación de conductas externalizantes de riesgo. |
| **Síntomas más Distintivos (Diferencia >15pp vs. Global)** | 1. **`pal001`:** Consumo de alcohol (+37pp)<br>2. **`pni001`:** Consumo de cigarrillos (+30pp)<br>3. **`psa007`:** Temor por daño a figura de apego (+28pp)<br>4. **`pmd018`:** Problemas de concentración (+21pp)<br>5. **`pmd016`:** Sentimientos de incapacidad (+21pp) | 1. **`pmd003`:** Sin irritabilidad (+17pp)<br>2. **`pad025`:** Sin conducta hiperactiva (+16pp)<br>3. **`pga028`:** Sin dolores de estómago (+15pp)<br>4. **`pad001`:** Sin problemas de atención (+15pp)<br>5. **`psa005`:** Sin necesidad de apego para dormir (+14pp) | 1. **`psa005`:** Necesidad de apego para dormir (+20pp)<br>2. **`pad025`:** Conducta hiperactiva inapropiada (+17pp)<br>3. **`pal001`:** Abstinencia total de alcohol (+15pp)<br>4. **`pni001`:** Abstinencia total de cigarrillos (+11pp)*<br>5. **`pga028`:** Dolores de estómago (+11pp)*<br>*(*Diferencias <15pp, pero entre las más altas para esta clase)* |
| **Riesgo Asociado (Dentro del espectro normativo)** | **Moderado.** Patrón que indica una mayor vulnerabilidad emocional y tendencia a la exploración de conductas de riesgo, lo que podría requerir más apoyo para una adaptación óptima. | **Muy Bajo.** Perfil que denota recursos personales y contextuales robustos para la regulación emocional y conductual, representando el estándar de bienestar dentro del grupo. | **Moderado.** Patrón que señala una mayor vulnerabilidad a la ansiedad y al malestar somático, con un estilo de afrontamiento cauteloso que podría limitar la exploración autónoma. |
| **Abordaje Sugerido (Enfoque preventivo y de promoción)** | **Refuerzo de habilidades de regulación.** Espacios para el desarrollo de estrategias de manejo emocional, toma de decisiones y educación sobre sustancias, en un marco de reducción de riesgos y no estigmatización. | **Fortalecimiento de factores protectores.** Programas de promoción de la salud mental dirigidos a mantener y generalizar los recursos de afrontamiento, autoestima y relaciones seguras que ya poseen. | **Apoyo a la autonomía y manejo de la ansiedad.** Estrategias para fomentar una independencia progresiva y segura, manejo de la preocupación excesiva y validación de las señales corporales, sin forzar la experimentación. |

## Variables más discriminatorias (k = 6)

| Variable                                | Cramér's V | p-value   | χ²  | Discriminación |
|-----------------------------------------|------------|-----------|-----|----------------|
| pcd012                                  | 0.6087     | 2.43e-75  | 359 | 1.000          |
| pal001                                  | 0.5774     | 1.31e-67  | 323 | 1.000          |
| pcd003                                  | 0.5154     | 1.65e-53  | 257 | 1.000          |
| pmj001                                  | 0.5042     | 3.85e-51  | 246 | 1.000          |
| pcd039                                  | 0.5016     | 1.34e-50  | 244 | 1.000          |
| pmd010                                  | 0.4795     | 4.27e-46  | 223 | 1.000          |
| pmd017                                  | 0.4079     | 6.00e-33  | 161 | 1.000          |
| pni001                                  | 0.4069     | 7.21e-63  | 320 | 0.996          |
| pmd015                                  | 0.3928     | 1.86e-30  | 149 | 1.000          |
| pmd012                                  | 0.3899     | 5.29e-30  | 147 | 1.000          |
| pmd016                                  | 0.3847     | 3.60e-29  | 143 | 1.000          |
| pcd013                                  | 0.3821     | 9.35e-29  | 141 | 1.000          |
| pmd006                                  | 0.3740     | 1.70e-27  | 135 | 1.000          |
| pmd003                                  | 0.3674     | 1.74e-26  | 131 | 1.000          |
| pad025                                  | 0.3598     | 2.38e-25  | 125 | 1.000          |
| pmd002                                  | 0.3589     | 3.21e-25  | 125 | 1.000          |
| pcd027                                  | 0.3583     | 3.96e-25  | 124 | 1.000          |
| psa007                                  | 0.3514     | 3.99e-24  | 120 | 1.000          |
| pod005                                  | 0.3409     | 1.22e-22  | 112 | 1.000          |
| pmd035                                  | 0.3406     | 1.19e-42  | 225 | 0.956          |
| pmd020                                  | 0.3397     | 1.82e-22  | 112 | 1.000          |
| pmd018                                  | 0.3395     | 1.93e-22  | 112 | 1.000          |
| pcd025                                  | 0.3370     | 4.17e-22  | 110 | 1.000          |
| pmd008                                  | 0.3335     | 1.26e-21  | 108 | 1.000          |
| pmd001                                  | 0.3297     | 4.14e-21  | 105 | 1.000          |

## Hallazgos principales:

### Variables más discriminantes (Cramér's V > 0.5):
- **`pcd012` – *"¿Alguna vez se metió en problemas por quedarse fuera 2 horas más allá del toque de queda?"***
    - **Cramér's V = 0.6087** 
- **`pal001` – *"¿Alguna vez, sin incluir sorbos de otros, ha tomado al menos una bebida alcohólica?"***
    - **Cramér's V = 0.5774**
- **`pcd003` – *"¿Alguna vez robó de alguien cuando no estaba/nadie miraba?"***
    - **Cramér's V = 0.5154**
- **`pmj001` – *"¿Alguna vez en toda su vida ha usado marihuana?"***
    - **Cramér's V = 0.5042**
- **`pcd039` – *"¿Alguna vez ha tenido problemas con la policía?"***
    - **Cramér's V = 0.5016**


### Dimensiones temáticas predominantes:

####  Conducta antisocial y delictiva grave:
Las variables más discriminantes en general se agrupan aquí, indicando que los comportamientos transgresores y el contacto con la ley son clave para definir clases.
- **Transgresiones y deshonestidad:**
    - `pcd012` – Meterse en problemas por violar el toque de queda.
    - `pcd003` – Robar en ausencia de la víctima.
    - `pcd039` – Tener problemas con la policía.
- **Agresión y crueldad:**
    - `pcd027` – Haber estado en una pelea física donde alguien salió/herir.
    - `pcd025` – Haber acosado (bullied) a alguien más pequeño.
    - `pcd023` – Haber sido físicamente cruel con un animal a propósito.

####  Consumo de sustancias:
Un segundo pilar fundamental para discriminar clases, con el alcohol y la marihuana como marcadores principales.
- `pal001` – Consumo de alcohol (al menos un trago completo).
- `pmj001` – Consumo de marihuana.
- `pni001` – Consumo de tabaco (fumado). (V = 0.4069)

####  Sintomatología internalizante  con enlentecimiento psicomotor:
- `pmd010` – Enlentecimiento psicomotor (hablar/caminar más lento).
- `pmd017` – Incapacidad para pensar tan rápido o claramente como de costumbre.
- `pmd015` – Autoculparse por cosas malas que sucedieron.
- `pmd012` – Menos energía.
- `pmd016` – Sentimientos de ineptitud/inferioridad.
- `pmd006` – Aumento significativo de peso.
- `pmd002` – Anhedonia/desinterés/tristeza.
- `pmd035` – Parecer triste o deprimido mucho tiempo.
- `pmd020` – Pensamientos sobre la muerte o morir.

####  Comportamiento oposicionista y disruptivo:
- `pod005` – Hacer cosas para molestar o enfadar a otros.
- `pod008` – Parecer enojado con personas o cosas.
- `pod007` – Malhumorado/fácilmente irritable.
- `pmd003` – A menudo malhumorado e irritable, con baja tolerancia a la frustración.

####  Síntomas de ansiedad (Separación/Preocupación):
- `psa007` – Miedo a que algo le pase a la figura de apego.
- `pga023` – Ser una persona tensa a la que le cuesta relajarse.
- `pga024` – Preocupación por haber cometido un error o hecho algo mal.

####  Síntomas de Inatención/Hiperactividad:
- `pad025` – Trepar/correr en momentos inapropiados.
- `pad022` – Hiperactividad motora.


## Categorías Sub y Sobre-Representadas

### Clase 0 (k = 6)
  
 403 observaciones (41.6%)
  
  ---
  Variables distintivas:

     1. psa007:
        SOBRE 1: 'NO' - 71% (287/403) vs 54% (522/968) global, +17pp
        SUB 1: 'SI' - 29% (116/403) vs 46% (446/968) global, -17pp

     2. pmd003:
        SOBRE 1: 'NO' - 78% (313/403) vs 61% (587/968) global, +17pp
        SUB 1: 'SI' - 22% (90/403) vs 39% (381/968) global, -17pp

     3. pmd018:
        SOBRE 1: 'NO' - 80% (323/403) vs 65% (631/968) global, +15pp
        SUB 1: 'SI' - 20% (80/403) vs 35% (337/968) global, -15pp

     4. pmd006:
        SOBRE 1: 'NO' - 100% (403/403) vs 85% (824/968) global, +15pp

     5. pal001:
        SOBRE 1: 'NO' - 100% (403/403) vs 85% (827/968) global, +15pp

     6. pmd017:
        SOBRE 1: 'NO' - 100% (403/403) vs 86% (831/968) global, +14pp

     7. pmd012:
        SOBRE 1: 'NO' - 95% (382/403) vs 81% (783/968) global, +14pp
        SUB 1: 'SI' - 5% (21/403) vs 19% (185/968) global, -14pp

     8. pmd001:
        SOBRE 1: 'NO' - 79% (319/403) vs 66% (635/968) global, +14pp
        SUB 1: 'SI' - 21% (84/403) vs 34% (333/968) global, -14pp

     9. pmd008:
        SOBRE 1: 'NO' - 87% (352/403) vs 74% (721/968) global, +13pp
        SUB 1: 'SI' - 13% (51/403) vs 26% (247/968) global, -13pp

    10. pod005:
        SOBRE 1: 'NO' - 100% (403/403) vs 88% (850/968) global, +12pp

    11. pmd016:
        SOBRE 1: 'NO' - 91% (365/403) vs 79% (763/968) global, +12pp
        SUB 1: 'SI' - 9% (38/403) vs 21% (205/968) global, -12pp

    12. pmd015:
        SOBRE 1: 'NO' - 95% (382/403) vs 84% (811/968) global, +11pp
        SUB 1: 'SI' - 5% (21/403) vs 16% (157/968) global, -11pp

    13. pni001:
        SOBRE 1: 'NO' - 100% (403/403) vs 89% (865/968) global, +11pp

    14. pmd002:
        SOBRE 1: 'NO' - 94% (380/403) vs 84% (815/968) global, +10pp
        SUB 1: 'SI' - 6% (23/403) vs 16% (153/968) global, -10pp

#### Perfil general:
La **Clase 0** representa el **subgrupo más grande (41.6%)** dentro de la muestra normativa y se caracteriza por un **perfil de bajo riesgo y alto ajuste**, con una **abstinencia de sustancias**, una **baja prevalencia de sintomatología depresiva e internalizante**, y una **mínima expresión de conductas oposicionistas**, mostrando el patrón de funcionamiento más adaptativo y resiliente en comparación con el perfil típico del grupo.

#### Hallazgos principales:

##### **Abstinencia total de sustancias:**
- **`pal001`**: 100% No ha tomado al menos un trago completo de alcohol (vs 85% global) → **+15pp**
- **`pni001`**: 100% No ha fumado cigarrillos (vs 89% global) → **+11pp**

##### **Baja sintomatología depresiva y cognitiva:**
- **`pmd006`**: 100% No aumentó mucho de peso en el último año (vs 85% global) → **+15pp**
- **`pmd017`**: 100% No tuvo problemas para pensar rápido o claramente (vs 86% global) → **+14pp**
- **`pmd012`**: 95% No tuvo menos energía (vs 81% global) → **+14pp**
- **`pmd001`**: 79% No tuvo un período en que pareció muy deprimido/alterado (vs 66% global) → **+13pp**
- **`pmd008`**: 87% No tuvo problemas de sueño (vs 74% global) → **+13pp**
- **`pmd016`**: 91% No se sintió incapaz o inferior (vs 79% global) → **+12pp**
- **`pmd015`**: 95% No se autoculpó por cosas malas (vs 84% global) → **+11pp**
- **`pmd002`**: 94% No se sintió desinteresado/triste (vs 84% global) → **+10pp**
- **`pmd003`**: 78% No estuvo malhumorado e irritable (vs 61% global) → **+17pp**
- **`pmd018`**: 80% No tuvo problemas para mantener la mente concentrada (vs 65% global) → **+15pp**

##### **Baja conducta oposicionista:**
- **`pod005`**: 100% No hizo cosas solo para molestar a otros (vs 88% global) → **+12pp**

##### **Menor ansiedad de separación:**
- **`psa007`**: 71% No tuvo miedo de que algo malo le pasara a su figura de apego (vs 54% global) → **+17pp**

#### Síntesis interpretativa:

La **Clase 0** constituye el **subgrupo normativo más prevalente (41.6%)**, caracterizado por un **patrón de alto ajuste y bajo riesgo**. Este perfil sugiere:

1.  **Abstinencia normativa:** Evitación completa del consumo de alcohol y tabaco, lo que refleja una alta adhesión a las normas sociales propias de la edad y, posiblemente, un entorno con mayor supervisión o valores conservadores respecto al consumo.
2.  **Bienestar emocional preservado:** Muy baja frecuencia de síntomas depresivos, cognitivos (lentitud, dificultad para concentrarse) y somáticos (problemas de sueño, cambios de peso). Esto indica una buena regulación emocional y un estado de ánimo estable dentro del rango normativo.
3.  **Baja irritabilidad y oposición:** Ausencia de conductas dirigidas a molestar a otros y una menor tendencia al mal humor, lo que apunta a un temperamento más fácil y relaciones interpersonales probablemente más armoniosas.
4.  **Seguridad en el apego:** Menor preocupación por la separación o el daño a las figuras de apego, lo que sugiere vínculos seguros y una adecuada autonomía para su etapa de desarrollo.

Este patrón refleja un **estilo de afrontamiento resiliente y adaptativo** dentro del espectro normal, donde la combinación de factores protectores conductuales (no consumo), emocionales (bajo estado de ánimo negativo) y relacionales (baja oposición y ansiedad) configura el perfil de mayor ajuste en la muestra. Representa la variante normativa con menor carga de dificultades internalizantes y externalizantes.

### Clase 1 (k = 6)
  
  46 observaciones (4.8%)
  
  ---
  Variables distintivas:

     1. pal001:
        SOBRE 1: 'SI' - 80% (37/46) vs 15% (141/968) global, +66pp
        SUB 1: 'NO' - 20% (9/46) vs 85% (827/968) global, -66pp

     2. pni001:
        SOBRE 1: 'SI' - 65% (30/46) vs 11% (102/968) global, +55pp
        SUB 1: 'NO' - 33% (15/46) vs 89% (865/968) global, -57pp

     3. pcd012:
        SOBRE 1: 'SI' - 48% (22/46) vs 3% (31/968) global, +45pp
        SUB 1: 'NO' - 52% (24/46) vs 97% (937/968) global, -45pp

     4. pmj001:
        SOBRE 1: 'SI' - 35% (16/46) vs 2% (22/968) global, +33pp
        SUB 1: 'NO' - 65% (30/46) vs 98% (946/968) global, -33pp

     5. pcd027:
        SOBRE 1: 'SI' - 37% (17/46) vs 6% (58/968) global, +31pp
        SUB 1: 'NO' - 63% (29/46) vs 94% (910/968) global, -31pp

     6. pcd039:
        SOBRE 1: 'SI' - 26% (12/46) vs 1% (12/968) global, +25pp
        SUB 1: 'NO' - 74% (34/46) vs 99% (956/968) global, -25pp

     7. psa007:
        SOBRE 1: 'SI' - 70% (32/46) vs 46% (446/968) global, +23pp
        SUB 1: 'NO' - 30% (14/46) vs 54% (522/968) global, -23pp

     8. pcd013:
        SOBRE 1: 'SI' - 15% (7/46) vs 1% (7/968) global, +14pp
        SUB 1: 'NO' - 85% (39/46) vs 99% (961/968) global, -14pp

     9. pmd008:
        SOBRE 1: 'SI' - 39% (18/46) vs 26% (247/968) global, +14pp
        SUB 1: 'NO' - 61% (28/46) vs 74% (721/968) global, -14pp

    10. pmd018:
        SOBRE 1: 'SI' - 48% (22/46) vs 35% (337/968) global, +13pp
        SUB 1: 'NO' - 52% (24/46) vs 65% (631/968) global, -13pp

    11. pmd001:
        SOBRE 1: 'SI' - 46% (21/46) vs 34% (333/968) global, +11pp
        SUB 1: 'NO' - 54% (25/46) vs 66% (635/968) global, -11pp

#### Perfil general:
La **Clase 1** representa un **subgrupo minoritario (4.8%)** dentro de la muestra normativa y se caracteriza por un **perfil de experimentación conductual temprana**, que combina un **significativo consumo de sustancias**, la **aparición de conductas transgresoras y antisociales**, junto con una **mayor carga de síntomas internalizantes**, mostrando un patrón de ajuste más complejo y de mayor riesgo relativo en comparación con la media del grupo normativo.

#### Hallazgos principales:

##### **Consumo temprano y significativo de sustancias:**
- **`pal001`**: 80% **Sí** ha tomado al menos un trago completo de alcohol (vs 15% global) → **+66pp**
- **`pni001`**: 65% **Sí** ha fumado cigarrillos (vs 11% global) → **+55pp**
- **`pmj001`**: 35% **Sí** ha usado marihuana (vs 2% global) → **+33pp**

##### **Conductas transgresoras y antisociales incipientes:**
- **`pcd012`**: 48% **Sí** se ha metido en problemas por violar el toque de queda (vs 3% global) → **+45pp**
- **`pcd027`**: 37% **Sí** ha estado en una pelea física con heridos o riesgo (vs 6% global) → **+31pp**
- **`pcd039`**: 26% **Sí** ha tenido problemas con la policía (vs 1% global) → **+25pp**
  
##### **Mayor sintomatología internalizante:**
- **`psa007`**: 70% **Sí** temió que algo malo le pasara a su figura de apego (vs 46% global) → **+23pp**
- **`pmd008`**: 39% **Sí** tuvo problemas de sueño (vs 26% global) → **+14pp**
- **`pmd018`**: 48% **Sí** tuvo problemas para concentrarse (vs 35% global) → **+13pp**
- **`pmd001`**: 46% **Sí** tuvo un período de aparente depresión/alteración (vs 34% global) → **+11pp**

#### Síntesis interpretativa:

La **Clase 1** constituye un **subgrupo normativo pequeño (4.8%) pero nítidamente diferenciado**, caracterizado por un **patrón de externalización temprana con comorbilidad internalizante**. Este perfil sugiere:

1.  **Inicio en conductas de riesgo:** El consumo de tabaco, alcohol y marihuana es alto para este subgrupo normativo
2.  **Externalización comportamental:** La presencia de peleas, problemas con la policía y violación de normas familiares (toque de queda) indica una tendencia a la búsqueda de sensaciones, el desafío a la autoridad y dificultades en el control de los impulsos, aunque dentro del rango de una muestra no clínica.
3.  **Malestar emocional coexistente:** A pesar del perfil externalizante, este grupo también reporta niveles más altos de ansiedad de separación, problemas de sueño y estado de ánimo bajo, lo que podría apuntar a un afrontamiento disfuncional del malestar interno o a factores de riesgo compartidos.
4.  **Posible vulnerabilidad dual:** La combinación de experimentación con sustancias, conductas disruptivas y síntomas emocionales sitúa a este subgrupo en una posición de mayor vulnerabilidad para el desarrollo de problemas más estructurados, aunque actualmente se encuentren dentro de la variación normativa.

Este patrón refleja una **variante normativa de alto riesgo relativo**, donde la convergencia de experimentación con sustancias, conductas transgresoras y malestar emocional señala a adolescentes que están probando límites y posiblemente utilizando la externalización como mecanismo de regulación. Representa un objetivo claro para intervenciones preventivas tempranas y de reducción de daños dentro de un contexto no clínico.
  
### Clase 2 (k = 6)

 
  190 observaciones (19.6%)
  
  ---
  Variables distintivas:

     1. pal001:
        SOBRE 1: 'SI' - 32% (60/190) vs 15% (141/968) global, +17pp
        SUB 1: 'NO' - 68% (130/190) vs 85% (827/968) global, -17pp

     2. pad025:
        SOBRE 1: 'NO' - 100% (190/190) vs 84% (815/968) global, +16pp

     3. pmd035:
        SOBRE 1: 'NO' - 100% (190/190) vs 87% (840/968) global, +13pp

     4. pmd020:
        SOBRE 1: 'NO' - 88% (168/190) vs 78% (756/968) global, +10pp
        SUB 1: 'SI' - 12% (22/190) vs 22% (212/968) global, -10pp

#### Perfil general:
La **Clase 2** representa un **subgrupo moderado (19.6%)** dentro de la muestra normativa y se caracteriza por un **perfil de excelente ajuste conductual y emocional**, con **ausencia total de conductas hiperactivas disruptivas**, una **muy baja sintomatología depresiva**, y una **experimentación limitada con sustancias** que, aunque superior al promedio global, se mantiene en un nivel moderado y no se asocia con otras conductas problemáticas, mostrando el patrón de funcionamiento más adaptativo después de la Clase 0.

#### Hallazgos principales:

##### **Excelente regulación conductual (ausencia de hiperactividad disruptiva):**
- **`pad025`**: 100% **No** trepaba en cosas/correra cuando no debía (vs 84% global) → **+16pp**

##### **Bienestar emocional destacado:**
- **`pmd035`**: 100% **No** parecía triste o deprimido mucho tiempo (vs 87% global) → **+13pp**
- **`pmd020`**: 88% **No** tuvo pensamientos sobre la muerte o morir (vs 78% global) → **+10pp**

##### **Experimentación moderada con alcohol:**
- **`pal001`**: 32% **Sí** ha tomado al menos un trago completo de alcohol (vs 15% global) → **+17pp**

#### Síntesis interpretativa:

La **Clase 2** constituye un **subgrupo normativo de muy buen ajuste (19.6%)**, caracterizado por un **patrón de regulación conductual ejemplar y bienestar emocional preservado**. Este perfil sugiere:

1.  **Control conductual excepcional:** La ausencia total de conductas hiperactivas e impulsivas inapropiadas (trepar/correr en momentos no permitidos) indica una capacidad sobresaliente para autorregular el comportamiento, respetar normas sociales y adaptarse a las demandas del entorno.
2.  **Fortaleza emocional:** La completa ausencia de apariencia depresiva prolongada y la baja frecuencia de pensamientos sobre la muerte reflejan un estado de ánimo estable, resiliencia emocional y probablemente buenas estrategias de afrontamiento.
3.  **Experimentación normativa limitada:** Aunque un tercio ha probado alcohol (más que el promedio global), este consumo no se acompaña de otras conductas de riesgo significativas (como consumo de otras sustancias, conductas antisociales o problemas emocionales graves), lo que sugiere una experimentación social ocasional dentro de parámetros normativos.

Este patrón refleja un **estilo de funcionamiento óptimo dentro del espectro normativo**, donde se combinan fortalezas en múltiples dominios: excelente control conductual, bienestar emocional y una experimentación con sustancias que, aunque presente, es moderada y no parece asociarse a deterioro. Representa una variante normativa con recursos protectores evidentes y un ajuste psicosocial muy adecuado para su etapa de desarrollo.
  
### Clase 3 (k = 6)

51 observaciones (5.3%)
  
  ---
  Variables distintivas:

     1. psa007:
        SOBRE 1: 'SI' - 78% (40/51) vs 46% (446/968) global, +32pp
        SUB 1: 'NO' - 22% (11/51) vs 54% (522/968) global, -32pp

     2. pad025:
        SOBRE 1: 'SI' - 47% (24/51) vs 16% (153/968) global, +31pp
        SUB 1: 'NO' - 53% (27/51) vs 84% (815/968) global, -31pp

     3. pcd003:
        SOBRE 1: 'SI' - 33% (17/51) vs 2% (22/968) global, +31pp
        SUB 1: 'NO' - 67% (34/51) vs 98% (946/968) global, -31pp

     4. pmd018:
        SOBRE 1: 'SI' - 61% (31/51) vs 35% (337/968) global, +26pp
        SUB 1: 'NO' - 39% (20/51) vs 65% (631/968) global, -26pp

     5. pmd016:
        SOBRE 1: 'SI' - 47% (24/51) vs 21% (205/968) global, +26pp
        SUB 1: 'NO' - 53% (27/51) vs 79% (763/968) global, -26pp

     6. pmd003:
        SOBRE 1: 'SI' - 65% (33/51) vs 39% (381/968) global, +25pp
        SUB 1: 'NO' - 35% (18/51) vs 61% (587/968) global, -25pp

     7. pcd025:
        SOBRE 1: 'SI' - 25% (13/51) vs 3% (32/968) global, +22pp
        SUB 1: 'NO' - 75% (38/51) vs 97% (936/968) global, -22pp

     8. pmd010:
        SOBRE 1: 'SI' - 29% (15/51) vs 8% (76/968) global, +22pp
        SUB 1: 'NO' - 71% (36/51) vs 92% (892/968) global, -22pp

     9. pmd001:
        SOBRE 1: 'SI' - 55% (28/51) vs 34% (333/968) global, +21pp
        SUB 1: 'NO' - 45% (23/51) vs 66% (635/968) global, -21pp

    10. pmd035:
        SOBRE 1: 'SI' - 24% (12/51) vs 11% (105/968) global, +13pp
        SUB 1: 'NO' - 67% (34/51) vs 87% (840/968) global, -20pp

    11. pmd020:
        SOBRE 1: 'SI' - 41% (21/51) vs 22% (212/968) global, +19pp
        SUB 1: 'NO' - 59% (30/51) vs 78% (756/968) global, -19pp

    12. pmd012:
        SOBRE 1: 'SI' - 37% (19/51) vs 19% (185/968) global, +18pp
        SUB 1: 'NO' - 63% (32/51) vs 81% (783/968) global, -18pp

    13. pmd008:
        SOBRE 1: 'SI' - 43% (22/51) vs 26% (247/968) global, +18pp
        SUB 1: 'NO' - 57% (29/51) vs 74% (721/968) global, -18pp

    14. pmd017:
        SOBRE 1: 'SI' - 31% (16/51) vs 14% (137/968) global, +17pp
        SUB 1: 'NO' - 69% (35/51) vs 86% (831/968) global, -17pp

    15. pmd002:
        SOBRE 1: 'SI' - 31% (16/51) vs 16% (153/968) global, +16pp
        SUB 1: 'NO' - 69% (35/51) vs 84% (815/968) global, -16pp

    16. pmd015:
        SOBRE 1: 'SI' - 31% (16/51) vs 16% (157/968) global, +15pp
        SUB 1: 'NO' - 69% (35/51) vs 84% (811/968) global, -15pp

    17. pcd012:
        SOBRE 1: 'SI' - 18% (9/51) vs 3% (31/968) global, +14pp
        SUB 1: 'NO' - 82% (42/51) vs 97% (937/968) global, -14pp

    18. pod005:
        SOBRE 1: 'SI' - 25% (13/51) vs 12% (118/968) global, +13pp
        SUB 1: 'NO' - 75% (38/51) vs 88% (850/968) global, -13pp

    19. pal001:
        SOBRE 1: 'NO' - 96% (49/51) vs 85% (827/968) global, +11pp

#### Perfil general:
La **Clase 3** representa un **subgrupo minoritario (5.3%)** dentro de la muestra normativa y se caracteriza por un **patrón multifacético de mayor dificultad**, que combina una **elevada ansiedad de separación**, una **notable presencia de síntomas depresivos y cognitivos**, **conductas hiperactivas**, y la **aparición de algunas conductas antisociales**, todo ello junto con una **marcada abstinencia de alcohol**, mostrando un perfil de ajuste más complejo y con mayores desafíos en múltiples dominios en comparación con la media del grupo.

#### Hallazgos principales:

##### **Alta ansiedad de separación:**
- **`psa007`**: 78%  temió que algo malo le pasara a su figura de apego (vs 46% global) → **+32pp**

##### **Sintomatología depresiva y cognitiva elevada:**
- **`pmd018`**: 61%  tuvo problemas para mantener la mente concentrada (vs 35% global) → **+26pp**
- **`pmd016`**: 47%  se sintió incapaz o inferior (vs 21% global) → **+26pp**
- **`pmd003`**: 65%  estuvo malhumorado e irritable (vs 39% global) → **+25pp**
- **`pmd010`**: 29%  parecía hacer cosas mucho más lento (vs 8% global) → **+22pp**
- **`pmd001`**: 55%  tuvo un período de aparente depresión/alteración (vs 34% global) → **+21pp**
- **`pmd020`**: 41%  tuvo pensamientos sobre la muerte o morir (vs 22% global) → **+19pp**
- **`pmd012`**: 37%  tuvo menos energía (vs 19% global) → **+18pp**
- **`pmd008`**: 43%  tuvo problemas de sueño (vs 26% global) → **+18pp**
- **`pmd017`**: 31%  tuvo problemas para pensar rápido o claramente (vs 14% global) → **+17pp**
- **`pmd002`**: 31%  se sintió desinteresado/triste (vs 16% global) → **+16pp**
- **`pmd015`**: 31%  se autoculpó por cosas malas (vs 16% global) → **+15pp**
- **`pmd035`**: 24%  parecía triste o deprimido mucho tiempo (vs 11% global) → **+13pp**

##### **Presencia de conductas hiperactivas y oposicionistas:**
- **`pad025`**: 47%  trepaba en cosas/corría cuando no debía (vs 16% global) → **+31pp**
- **`pod005`**: 25%  hizo cosas solo para molestar a otros (vs 12% global) → **+13pp**

##### **Aparición de conductas antisociales:**
- **`pcd003`**: 33%  robó de alguien cuando no estaba/nadie miraba (vs 2% global) → **+31pp**
- **`pcd025`**: 25%  acosó (bullied) a alguien más pequeño (vs 3% global) → **+22pp**
- **`pcd012`**: 18%  se metió en problemas por violar el toque de queda (vs 3% global) → **+14pp**

##### **Abstinencia de alcohol:**
- **`pal001`**: 96% **No** ha tomado al menos un trago completo de alcohol (vs 85% global) → **+11pp**

#### Síntesis interpretativa:

La **Clase 3** constituye un **subgrupo normativo pequeño (5.3%) con un perfil de vulnerabilidad amplia**, caracterizado por **desafíos internalizantes y externalizantes**. Este perfil sugiere:

1.  **Angustia internalizante prominente:** Este grupo reporta niveles consistentemente más altos en casi todos los síntomas internalizantes evaluados (estado de ánimo, cognición, psicomotricidad, somatización), lo que indica un malestar emocional significativo y difuso dentro del rango normativo.
2.  **Inquietud y dificultades de regulación conductual:** La presencia de hiperactividad motora inapropiada y conductas oposicionistas sugiere desafíos en el control de los impulsos y la autorregulación comportamental.
3.  **Externalización en forma de transgresiones:** La aparición de conductas antisociales como robo y acoso, aunque en una minoría del subgrupo, marca una clara desviación de las normas sociales en comparación con el resto de la muestra normativa.
4.  **Apego ansioso y evitación del riesgo de sustancias:** La elevada ansiedad por separación coexiste con una abstinencia casi total de alcohol, lo que podría reflejar un patrón de afrontamiento cauteloso o inhibido frente a ciertos riesgos, a pesar de la presencia de otros comportamientos desafiantes.

Este patrón refleja una **variante normativa de complejidad elevada**, donde convergen múltiples dominios de dificultad (emocional, conductual, social) sin que el consumo de alcohol aparezca como un mecanismo de afrontamiento. Representa un subgrupo dentro de la variación normal que podría beneficiarse de apoyo para la regulación emocional y conductual, manteniendo al mismo tiempo ciertos factores protectores como la abstinencia de sustancias.

### Clase 4 (k = 6)


  90 observaciones (9.3%)
  
  ---
  Variables distintivas:

     1. pmd035:
        SOBRE 1: 'SI' - 40% (36/90) vs 11% (105/968) global, +29pp
        SOBRE 2: 'nan' - 13% (12/90) vs 2% (23/968) global, +11pp
        SUB 1: 'NO' - 47% (42/90) vs 87% (840/968) global, -40pp

     2. pmd016:
        SOBRE 1: 'SI' - 59% (53/90) vs 21% (205/968) global, +38pp
        SUB 1: 'NO' - 41% (37/90) vs 79% (763/968) global, -38pp

     3. pmd012:
        SOBRE 1: 'SI' - 56% (50/90) vs 19% (185/968) global, +36pp
        SUB 1: 'NO' - 44% (40/90) vs 81% (783/968) global, -36pp

     4. pmd015:
        SOBRE 1: 'SI' - 51% (46/90) vs 16% (157/968) global, +35pp
        SUB 1: 'NO' - 49% (44/90) vs 84% (811/968) global, -35pp

     5. pmd020:
        SOBRE 1: 'SI' - 56% (50/90) vs 22% (212/968) global, +34pp
        SUB 1: 'NO' - 44% (40/90) vs 78% (756/968) global, -34pp

     6. pmd001:
        SOBRE 1: 'SI' - 68% (61/90) vs 34% (333/968) global, +33pp
        SUB 1: 'NO' - 32% (29/90) vs 66% (635/968) global, -33pp

     7. pmd008:
        SOBRE 1: 'SI' - 59% (53/90) vs 26% (247/968) global, +33pp
        SUB 1: 'NO' - 41% (37/90) vs 74% (721/968) global, -33pp

     8. pmd010:
        SOBRE 1: 'SI' - 41% (37/90) vs 8% (76/968) global, +33pp
        SUB 1: 'NO' - 59% (53/90) vs 92% (892/968) global, -33pp

     9. psa007:
        SOBRE 1: 'SI' - 78% (70/90) vs 46% (446/968) global, +32pp
        SUB 1: 'NO' - 22% (20/90) vs 54% (522/968) global, -32pp

    10. pmd017:
        SOBRE 1: 'SI' - 46% (41/90) vs 14% (137/968) global, +31pp
        SUB 1: 'NO' - 54% (49/90) vs 86% (831/968) global, -31pp

    11. pmd002:
        SOBRE 1: 'SI' - 47% (42/90) vs 16% (153/968) global, +31pp
        SUB 1: 'NO' - 53% (48/90) vs 84% (815/968) global, -31pp

    12. pmd003:
        SOBRE 1: 'SI' - 69% (62/90) vs 39% (381/968) global, +30pp
        SUB 1: 'NO' - 31% (28/90) vs 61% (587/968) global, -30pp

    13. pni001:
        SOBRE 1: 'SI' - 37% (33/90) vs 11% (102/968) global, +26pp
        SUB 1: 'NO' - 63% (57/90) vs 89% (865/968) global, -26pp

    14. pmd018:
        SOBRE 1: 'SI' - 57% (51/90) vs 35% (337/968) global, +22pp
        SUB 1: 'NO' - 43% (39/90) vs 65% (631/968) global, -22pp

    15. pmd006:
        SOBRE 1: 'SI' - 37% (33/90) vs 15% (144/968) global, +22pp
        SUB 1: 'NO' - 63% (57/90) vs 85% (824/968) global, -22pp

    16. pal001:
        SOBRE 1: 'SI' - 36% (32/90) vs 15% (141/968) global, +21pp
        SUB 1: 'NO' - 64% (58/90) vs 85% (827/968) global, -21pp

#### Perfil general:
La **Clase 4** representa un **subgrupo pequeño (9.3%)** dentro de la muestra normativa y se caracteriza por un **perfil de malestar emocional multifacético y prominente**, que combina una **elevada carga de síntomas depresivos en múltiples dimensiones** (afectiva, cognitiva, psicomotora y somática), una **fuerte ansiedad de separación**, junto con una **experimentación moderada con tabaco y alcohol**.

#### Hallazgos principales:

##### **Sintomatología depresiva grave y generalizada:**
- **`pmd001`**: 68%  tuvo un período de aparente depresión/alteración (vs 34% global) → **+33pp**
- **`pmd003`**: 69%  estuvo malhumorado e irritable (vs 39% global) → **+30pp**
- **`pmd016`**: 59%  se sintió incapaz o inferior (vs 21% global) → **+38pp**
- **`pmd012`**: 56%  tuvo menos energía (vs 19% global) → **+36pp**
- **`pmd020`**: 56%  tuvo pensamientos sobre la muerte o morir (vs 22% global) → **+34pp**
- **`pmd008`**: 59%  tuvo problemas de sueño (vs 26% global) → **+33pp**
- **`pmd015`**: 51%  se autoculpó por cosas malas (vs 16% global) → **+35pp**
- **`pmd010`**: 41%  parecía hacer cosas mucho más lento (vs 8% global) → **+33pp**
- **`pmd002`**: 47%  se sintió desinteresado/triste (vs 16% global) → **+31pp**
- **`pmd017`**: 46%  tuvo problemas para pensar rápido o claramente (vs 14% global) → **+31pp**
- **`pmd035`**: 40%  parecía triste o deprimido mucho tiempo (vs 11% global) → **+29pp**
- **`pmd018`**: 57%  tuvo problemas para mantener la mente concentrada (vs 35% global) → **+22pp**
- **`pmd006`**: 37%  aumentó mucho de peso (vs 15% global) → **+22pp**

##### **Alta ansiedad de separación:**
- **`psa007`**: 78%  temió que algo malo le pasara a su figura de apego (vs 46% global) → **+32pp**

##### **Experimentación moderada con sustancias:**
- **`pni001`**: 37%  ha fumado cigarrillos (vs 11% global) → **+26pp**
- **`pal001`**: 36%  ha tomado al menos un trago completo de alcohol (vs 15% global) → **+21pp**

#### Síntesis interpretativa:

La **Clase 4** constituye un **subgrupo normativo con un perfil de vulnerabilidad emocional destacada (9.3%)**, caracterizado por **síntomas internalizantes que abarcan el afecto, la cognición, la motivación y las funciones somáticas**. Este perfil sugiere:

1.  **Malestar emocional multifacético:** Este grupo reporta niveles muy elevados en prácticamente todos los indicadores internalizantes evaluados, desde la tristeza y la irritabilidad hasta el enlentecimiento psicomotor, los problemas de sueño, la fatiga y los pensamientos de muerte. Esto indica una experiencia subjetiva de sufrimiento emocional significativa y generalizada dentro del rango normativo.
2.  **Dificultades cognitivas y motivacionales:** Los problemas de concentración, la indecisión, la autocrítica severa y la pérdida de energía sugieren un impacto notable en el funcionamiento diario y la percepción de autoeficacia.
3.  **Apego ansioso:** La alta preocupación por la seguridad de la figura de apego apunta a inseguridades en los vínculos o a una elevada sensibilidad a la separación, que puede ser tanto un factor contribuyente como una consecuencia del malestar emocional.
4.  **Consumo de sustancias como posible correlato:** La experimentación con tabaco y alcohol, aunque moderada, es superior al promedio. 

Este patrón refleja la **variante normativa con la mayor carga de sintomatología internalizante**, donde el malestar emocional es el rasgo definitorio. Representa un subgrupo que, aunque dentro de los límites de la muestra no clínica, experimenta un grado notable de sufrimiento psicológico y podría beneficiarse más de recursos de apoyo emocional y estrategias de afrontamiento saludable. La coexistencia de ansiedad de separación y consumo de sustancias añade complejidad a este perfil de vulnerabilidad.
  
### Clase 5 (k = 6)


 188 observaciones (19.4%)
  
  ---
  Variables distintivas:

     1. pmd003:
        SOBRE 1: 'SI' - 60% (112/188) vs 39% (381/968) global, +20pp
        SUB 1: 'NO' - 40% (76/188) vs 61% (587/968) global, -20pp

     2. pmd018:
        SOBRE 1: 'SI' - 54% (101/188) vs 35% (337/968) global, +19pp
        SUB 1: 'NO' - 46% (87/188) vs 65% (631/968) global, -19pp

     3. pad025:
        SOBRE 1: 'SI' - 34% (63/188) vs 16% (153/968) global, +18pp
        SUB 1: 'NO' - 66% (125/188) vs 84% (815/968) global, -18pp

     4. pod005:
        SOBRE 1: 'SI' - 28% (52/188) vs 12% (118/968) global, +15pp
        SUB 1: 'NO' - 72% (136/188) vs 88% (850/968) global, -15pp

     5. pmd001:
        SOBRE 1: 'SI' - 46% (87/188) vs 34% (333/968) global, +12pp
        SUB 1: 'NO' - 54% (101/188) vs 66% (635/968) global, -12pp

     6. pmd006:
        SOBRE 1: 'SI' - 27% (50/188) vs 15% (144/968) global, +12pp
        SUB 1: 'NO' - 73% (138/188) vs 85% (824/968) global, -12pp

     7. pni001:
        SOBRE 1: 'NO' - 100% (188/188) vs 89% (865/968) global, +11pp

#### Perfil general:
La **Clase 5** representa un **subgrupo considerable (19.4%)** dentro de la muestra normativa y se caracteriza por un **perfil de malestar emocional y regulación conductual moderados**, que combina una **elevada irritabilidad y dificultades de concentración**, la **presencia de conductas hiperactivas y oposicionistas leves**, junto con una **abstinencia total de tabaco**, mostrando un patrón de ajuste intermedio donde los desafíos internalizantes y externalizantes coexisten sin la presencia de conductas antisociales graves o consumo problemático de sustancias.

#### Hallazgos principales:

##### **Sintomatología depresiva y cognitiva moderada:**
- **`pmd003`**: 60% **SÍ** estuvo malhumorado e irritable (vs 39% global) → **+20pp**
- **`pmd018`**: 54% **SÍ** tuvo problemas para mantener la mente concentrada (vs 35% global) → **+19pp**
- **`pmd001`**: 46% **SÍ** tuvo un período de aparente depresión/alteración (vs 34% global) → **+12pp**
- **`pmd006`**: 27% **SÍ** aumentó mucho de peso (vs 15% global) → **+12pp**

##### **Conductas hiperactivas y oposicionistas leves:**
- **`pad025`**: 34% **SÍ** trepaba en cosas/correra cuando no debía (vs 16% global) → **+18pp**
- **`pod005`**: 28% **SÍ** hizo cosas solo para molestar a otros (vs 12% global) → **+15pp**

##### **Abstinencia de tabaco:**
- **`pni001`**: 100% **NO** ha fumado cigarrillos (vs 89% global) → **+11pp**

#### Síntesis interpretativa:

La **Clase 5** constituye un **subgrupo normativo prevalente (19.4%) con un perfil mixto de dificultades leves a moderadas**, caracterizado por una **combinación de síntomas internalizantes y externalizantes de baja severidad**. Este perfil sugiere:

1.  **Desafíos en la regulación emocional y atencional:** La elevada irritabilidad y los problemas de concentración son los rasgos más destacados, indicando posibles dificultades en la gestión de la frustración y en el mantenimiento del foco atencional, dentro de los límites normativos.
2.  **Inquietud motora y tendencias oposicionistas:** La presencia de hiperactividad motora inapropiada y conductas dirigidas a molestar a otros refleja una dificultad leve en el control de los impulsos y en el seguimiento de normas sociales, sin escalar a conductas agresivas o antisociales graves.
3.  **Abstinencia como posible factor protector o característica distintiva:** La abstinencia total del consumo de tabaco, en contraste con la elevación en otros dominios, puede indicar una aversión al riesgo específica para esta sustancia, un entorno con mayores controles, o un estilo de afrontamiento que no incluye el uso de esta droga como regulador del malestar.
4.  **Perfil de internalización externalizante:** La combinación de irritabilidad (internalizante) con hiperactividad y oposición (externalizante) sugiere un patrón donde el malestar emocional se expresa parcialmente a través de la conducta, pero de una manera contenida y no severa.

Este patrón refleja una **variante normativa común donde coexisten desafíos emocionales y conductuales de baja intensidad**. Representa un subgrupo que puede experimentar un estrés o malestar subclínico significativo, manifestado principalmente como irritabilidad, desatención e inquietud, pero que mantiene un buen control sobre conductas de riesgo más graves (como el consumo de tabaco o actos antisociales). Es un perfil que podría beneficiarse de estrategias para mejorar la regulación emocional y el manejo de la frustración.

## Síntesis Comparativa de las 6 Clases Normativas

| **Criterio** | **Clase 0: Ajuste Óptimo** | **Clase 1: Externalización Temprana** | **Clase 2: Regulación Ejemplar** | **Clase 3: Vulnerabilidad Amplia** | **Clase 4: Malestar Emocional Prominente** | **Clase 5: Dificultades Mixtas Leves** |
| :--- | :--- | :--- | :--- | :--- | :--- | :--- |
| **Tamaño** | 41.6% (403) | 4.8% (46) | 19.6% (190) | 5.3% (51) | 9.3% (90) | 19.4% (188) |
| **Patrón Principal** | **Bajo riesgo y alto ajuste.** Abstinencia de sustancias y muy baja sintomatología emocional y conductual problemática. | **Experimentación conductual temprana.** Consumo significativo de sustancias y aparición de conductas transgresoras, con malestar emocional coexistente. | **Excelente regulación conductual y bienestar emocional.** Ausencia de hiperactividad disruptiva y sintomatología depresiva, con experimentación social limitada. | **Multifacético con desafíos internalizantes y externalizantes.** Alta ansiedad de separación, síntomas depresivos, hiperactividad y conductas antisociales leves, con abstinencia de alcohol. | **Malestar emocional multifacético y prominente.** Carga elevada de síntomas depresivos en todas las dimensiones y alta ansiedad de separación, con experimentación moderada en sustancias. | **Irritabilidad y regulación conductual moderada.** Combinación de síntomas internalizantes (irritabilidad, desatención) y externalizantes leves (hiperactividad, oposición), con abstinencia de tabaco. |
| **Núcleo del Perfil** | **Resiliencia y adherencia normativa.** Caracterizado por la **sub-representación** de casi todos los problemas evaluados. Representa el perfil de mayor ajuste dentro de la variación normal. | **Externalización con comorbilidad internalizante.** La experimentación con sustancias y el desafío a normas se combinan con ansiedad de separación y síntomas depresivos, señalando un perfil de mayor riesgo relativo. | **Fortalezas protectoras.** Control conductual (sin hiperactividad inapropiada) y estado de ánimo estable, lo que permite una experimentación social acotada sin consecuencias aparentes. | **Complejidad normativa.** Conglomerado de vulnerabilidades emocionales (apego ansioso, depresión), regulatorias (impulsividad motora) y sociales (transgresiones), pero con evitación del alcohol. | **Sufrimiento psicológico subclínico.** La angustia internalizante es el rasgo dominante, afectando el afecto, la cognición, la motivación y las funciones somáticas, con consumo de sustancias como posible correlato. | **Malestar externalizante.** La irritabilidad y los problemas atencionales son el núcleo, expresándose parcialmente a través de conductas inquietas y oposicionistas leves, pero con control sobre riesgos mayores. |
| **Variables más Distintivas** | 1. **pal001:** Abstinencia de alcohol (+15pp en 'NO')<br>2. **pmd006:** Sin aumento de peso importante (+15pp en 'NO')<br>3. **pmd017:** Sin lentitud cognitiva (+14pp en 'NO') | 1. **pal001:** Consumo de alcohol (+66pp en 'SÍ')<br>2. **pni001:** Consumo de tabaco (+55pp en 'SÍ')<br>3. **pcd012:** Problemas por toque de queda (+45pp en 'SÍ') | 1. **pad025:** Sin hiperactividad motora inapropiada (+16pp en 'NO')<br>2. **pmd035:** Sin apariencia depresiva (+13pp en 'NO')<br>3. **pal001:** Consumo de alcohol (+17pp en 'SÍ') | 1. **psa007:** Ansiedad por separación (+32pp en 'SÍ')<br>2. **pad025:** Hiperactividad motora (+31pp en 'SÍ')<br>3. **pcd003:** Robo en ausencia (+31pp en 'SÍ') | 1. **pmd016:** Sentimientos de inferioridad (+38pp en 'SÍ')<br>2. **pmd012:** Falta de energía (+36pp en 'SÍ')<br>3. **pmd015:** Autoculpa (+35pp en 'SÍ') | 1. **pmd003:** Irritabilidad (+20pp en 'SÍ')<br>2. **pmd018:** Problemas de concentración (+19pp en 'SÍ')<br>3. **pad025:** Hiperactividad motora (+18pp en 'SÍ') |
| **Riesgo Asociado** | **Muy Bajo.** Representa el estándar de ajuste saludable dentro de la muestra. El riesgo es mínimo, asociado únicamente a la posible transición hacia otros perfiles en contextos de alto estrés. | **Moderado-Alto.** Mayor vulnerabilidad para la consolidación de patrones de consumo problemático, conductas antisociales y para el agravamiento del malestar emocional. | **Muy Bajo.** Fortalezas conductuales y emocionales evidentes. El riesgo se limita a la posible normalización de un consumo alcohol que, aunque actualmente moderado, podría incrementarse. | **Moderado .** Vulnerabilidad en múltiples frentes (emocional, conductual, social) que podría traducirse en deterioro funcional o en la cronificación de síntomas si no se manejan adecuadamente. | **Moderado-Alto.** La intensidad y generalización del malestar emocional sugiere un mayor sufrimiento psicológico y potencial para un deterioro en el funcionamiento diario y las relaciones. | **Bajo-Moderado .** La presencia constante de irritabilidad y desatención, junto con conductas disruptivas leves, indica un desgaste en la regulación que podría afectar el rendimiento y las relaciones. |
| **Enfoque Sugerido** | **Promoción de la salud y prevención universal.** Refuerzo de los factores protectores existentes (apego seguro, habilidades sociales, valores de autocuidado) para mantener la trayectoria positiva. | **Prevención indicada y reducción de daños.** Orientación sobre los riesgos del consumo temprano y de las conductas transgresoras, combinada con apoyo para la regulación emocional y el desarrollo de habilidades prosociales. | **Refuerzo de las fortalezas y educación sobre riesgos.** Validación y potenciación de sus habilidades de regulación. Educación sobre los efectos del alcohol para que la experimentación se mantenga dentro de límites seguros. | **Apoyo integral y focalizado.** Estrategias para manejar la ansiedad, mejorar la regulación emocional y conductual, y fomentar habilidades sociales alternativas a las conductas antisociales, aprovechando la abstinencia de alcohol como factor protector. | **Apoyo emocional y psicoeducación prioritaria.** Espacios para el procesamiento del malestar emocional, desarrollo de estrategias de afrontamiento saludables y psicoeducación sobre la relación entre estado de ánimo y consumo de sustancias. | **Fortalecimiento de habilidades de regulación.** Entrenamiento en manejo de la irritabilidad, técnicas para mejorar la concentración y alternativas para canalizar la inquietud motora, aprovechando la abstinencia de tabaco. |


In [15]:
guardar_df(final_df, 'LCA_1558', 'LCA')
guardar_df(final_df_norm, 'LCA_norm', 'LCA')
guardar_df(prob_df, 'LCA_prob_1558', 'LCA_prob')
guardar_df(prob_df_norm, 'LCA_prob_norm', 'LCA_prob')

✓ DataFrame guardado como:
  • CSV: C:\Users\Lan_1\Documents\VictorL_TesinaMACI\VictorL_TesinaMACI\Python\resultados\LCA\csv\LCA_1558.csv
  • Excel: C:\Users\Lan_1\Documents\VictorL_TesinaMACI\VictorL_TesinaMACI\Python\resultados\LCA\excel\LCA_1558.xlsx
✓ DataFrame guardado como:
  • CSV: C:\Users\Lan_1\Documents\VictorL_TesinaMACI\VictorL_TesinaMACI\Python\resultados\LCA\csv\LCA_norm.csv
  • Excel: C:\Users\Lan_1\Documents\VictorL_TesinaMACI\VictorL_TesinaMACI\Python\resultados\LCA\excel\LCA_norm.xlsx
✓ DataFrame guardado como:
  • CSV: C:\Users\Lan_1\Documents\VictorL_TesinaMACI\VictorL_TesinaMACI\Python\resultados\LCA_prob\csv\LCA_prob_1558.csv
  • Excel: C:\Users\Lan_1\Documents\VictorL_TesinaMACI\VictorL_TesinaMACI\Python\resultados\LCA_prob\excel\LCA_prob_1558.xlsx
✓ DataFrame guardado como:
  • CSV: C:\Users\Lan_1\Documents\VictorL_TesinaMACI\VictorL_TesinaMACI\Python\resultados\LCA_prob\csv\LCA_prob_norm.csv
  • Excel: C:\Users\Lan_1\Documents\VictorL_TesinaMACI\VictorL_TesinaM